src/sidebar/components/sidebar-content-omega.js

```diff
@@ -0,0 +1,168 @@
+import { createElement } from 'preact';
+import propTypes from 'prop-types';
+import { useEffect, useRef } from 'preact/hooks';
+
+import { withServices } from '../util/service-context';
+import useStore from '../store/use-store';
+import { tabForAnnotation } from '../util/tabs';
+
+import FocusedModeHeader from './focused-mode-header';
+import LoggedOutMessage from './logged-out-message';
+import LoginPromptPanel from './login-prompt-panel';
+import SearchStatusBar from './search-status-bar';
+import SelectionTabs from './selection-tabs';
+import SidebarContentError from './sidebar-content-error';
+import ThreadList from './thread-list';
+
+/**
+ * Render the sidebar and its components
+ */
+function SidebarContent({
+  frameSync,
+  onLogin,
+  onSignUp,
+  loadAnnotationsService,
+  rootThread: rootThreadService,
+  streamer,
+}) {
+  const rootThread = useStore(store =>
+    rootThreadService.thread(store.getState())
+  );
+
+  // store state values
+  const focusedGroupId = useStore(store => store.focusedGroupId());
+  const hasAppliedFilter = useStore(store => store.hasAppliedFilter());
+  const isFocusedMode = useStore(store => store.focusModeEnabled());
+  const isLoading = useStore(store => {
+    return !store.hasFetchedAnnotations() || store.isFetchingAnnotations();
```

Comment:
There's a state of "haven't started loading yet" that is relevant in this component. This prevents a premature flash of, e.g., `<SidebarContentError>` when the fetch request for the relevant annotation(s) hasn't yet been dispatched.

Reply:
I think we might need a more specific name than `isLoading` here, because there may be cases where some non-annotation loading activity is happening, but this value remains `false` because annotations have already been fetched.

Reply:
Yes, I think you are correct. I had already been hemming and hawing about naming.